sheetstorm/urls.py

```diff
@@ -18,7 +18,7 @@
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^', include('users.urls')),
-    url(r'^managers/', include('managers.urls')),
+    url(r'^', include('managers.urls')),
```

Comment:
I think this should stay, as we discussed on project code review, we should keep endpoints separated.

Reply:
Done